src/Panes/CerberePane.vala

```diff
@@ -25,7 +25,7 @@ namespace ElementaryTweaks {
         }
 
         construct {
-            if (Util.schema_exists ("org.pantheon.desktop.cerbere")) {
+            if (Util.schema_exists ("io.elementary.desktop.cerbere")) {
```

Comment:
I suppose we can change to `if (Util.schema_exists ("org.pantheon.desktop.cerbere") || Util.schema_exists ("io.elementary.desktop.cerbere")) {`